cairo/src/word.rs

```diff
@@ -0,0 +1,429 @@
+//! The Cairo language works natively for field elements in the finite field with
+//! modulus 0x800000000000011000000000000000000000000000000000000000000000001
+//! This is the hexadecimal value for 2 ^ 251 + 17 * 2 ^ 192 + 1
+//! Our Pallas curves have 256 bits, so Cairo native instructions will fit.
```

Comment:
I thought the pallas curves had 255 bits :o

Reply:
Oh well they do, I meant that they were large enough to fit the Cairo modulus. I'll write it more precisely.